src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -33,21 +36,58 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
-
+    
+    // Set default variables to create UserAuthenticationData object.
     Boolean loggedIn = false;
+    String email = "";
```

Comment:
Was there a reason why this was renamed from userEmail to email? It doesn't make a huge difference but userEmail seems more descriptive.

Reply:
I had some variables named email and others named userEmail across different files. I wanted to change all of them to the same name, and went for just email to stay consistent with the database, but I could change to userEmail if you think it's better!

Reply:
Ah okay that's fine, that makes sense! Doesn't really matter, I was mainly just curious why it was changed.